src/components/SecondaryPanes/Breakpoints.css

```diff
@@ -62,6 +95,10 @@ html .breakpoints-list .breakpoint.paused {
   display: inline-block;
   padding-inline-start: 2px;
   cursor: default;
+  flex-grow: 1;
+  text-overflow: ellipsis;
+  overflow: hidden;
```

Comment:
This CSS ensures that the breakpoint code gets the most space but is also ellipsis'd